instructables/configuring_rbac.rst

```diff
@@ -0,0 +1,148 @@
+Configuring RBAC
+================
+
+StackStorm enterprise edition brings support for Role Based Access Control(RBAC). RBAC allows a StackStorm administrator
+control over what functions are accessible to a user within StackStorm.
+
+**Possible scenarios :**
+
+1. A user owns a pack i.e is able to view, create, delete, modify and where applicable execute various resources like actions, rules, sensors.
+2. A user can create rules, execute actions and view a handful of actions.
+3. A user capable of viewing actions in a pack but cannot execute any action.
+
+This guide provides a walk-through of scenario 1 i.e configuring a user as a pack owner. The steps to be followed are by an
+Administrator of StackStorm on a box that is running StackStorm.
+
+User creation
+-------------
+All user and password management is kept outside of StackStorm. Documentation on Authentication_ describe how to
+confirgure StackStorm with various identity providers.
+
+.. _Authentication: http://docs.stackstorm.com/latest/config/authentication.html
+
+For sake of this guide let us assume that the identify provider is managed by the OS on which StackStorm runs.
+
+To create a user and set-up a password on most linux systems -
+
+.. sourcecode:: bash
+
+    $ useradd rbacu1
+    $ passwd rbacu1
+
+Once this user is created StackStorm will allow access to this user. (Optional) To validate try -
+
+.. sourcecode:: bash
+
+    $ st2 auth rbacu1 -p <RBACU1_PASSWORD>
+    $ export ST2_AUTH_TOKEN=<USER_SCOPED_AUTH_TOKEN>
+    $ st2 action list
+
+Role creation
+-------------
+A newly created user has no assigned permissions. Each permission must be explicitly assigned to a user. To assign
+permission grants StackStorm requires creation of a role and then associating this role with a user. In this case we are trying to create a pack owner role.
+
+Lets first make sure there is a pack `x` we can use to experiment.
+
+.. sourcecode:: bash
+
+    $ cd /opt/stackstorm/packs/
+    $ mkdir x
```

Comment:
Yeah x is a placeholder. On the other hand using a real name is hard as I want to showcase action create and rule create - to do that with something like libcloud gets messy since I am moving actions and rule into the pack etc.